CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/uncollide/experimental/Dimensions.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2010-2020 Australian Signals Directorate
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.plugins.arrangements.uncollide.experimental;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * @author Nova
+ */
+enum Dimensions {
```

Comment:
Add access modifier here

Reply:
Made public as it is a parameter for the plugin which must be public so that it can be registered and called from other packages.